php-classes/Slate/CBL/CompetenciesRequestHandler.php

```diff
@@ -171,13 +171,21 @@ public static function handleDemonstrationsRequest(Competency $Competency)
 
         // query demonstrations sums
         try {
-            $skillDemonstrations = DB::allRecords(
-                'SELECT Demonstration.ID AS DemonstrationID, Demonstration.StudentID, DemonstrationSkill.SkillID, DemonstrationSkill.Level'
-                .' FROM (SELECT ID FROM `%s` WHERE CompetencyID = %u) AS Skill'
-                .' JOIN `%s` DemonstrationSkill'
-                .'  ON DemonstrationSkill.SkillID = Skill.ID'
-                .' JOIN (SELECT ID, StudentID FROM `%s` WHERE StudentID IN (%s)) Demonstration'
-                .'  ON Demonstration.ID = DemonstrationSkill.DemonstrationID'
+            $skillDemonstrations = DB::allRecords('
```

Comment:
This is how our DBA liked to format SQL, I think it's much more readable, not that the last character of the verb lines up rather than the first. ``AS``, ``ON`` and ``IN`` typically don't wrap to their own line unless it's getting too long.